lib/tasks/db.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+if Rake::Task['db:schema:dump:clickhouse'].present?
+  Rake::Task['db:schema:dump:clickhouse'].enhance do
+    Rake::Task['db:clickhouse:filter'].invoke
+  end
+end
+
+namespace :db do |args|
+  desc 'Filter secrets from clickhouse schema file'
+  task 'clickhouse:filter' do
```

Comment:
Rails/RakeEnvironment: Include `:environment` task as a dependency for all Rake tasks.